src/server/utils/logging/development-formatters/friendly.js

```diff
@@ -36,7 +36,7 @@ function createSpinner(opts) {
 const oneAppDevCdnSpinner = createSpinner({
   text: 'Starting one-app-dev-cdn server...',
   spinner: 'bouncingBar',
```

Comment:
Would you need to add the option `isEnabled: false`?

Reply:
I am not sure, would like to get input from @10xLaCroixDrinker or @mtomcal on this part